index.html

```diff
@@ -112,27 +114,31 @@ <h5 class="cardtitle fw-bolder mx-2 text-center opacity-100">Network</h5>
       </div>
 
       <div class="card register-btn h-50 p-2 mx-auto d-flex flex-row justify-content-evenly align-items-center
-                rounded-0 mb-3">
-        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light text-decoration-none
+                rounded-0  my-4 ">
+        <a class="cardtitle fw-bold py-3 text-center mx-3 opacity-100 text-light
           " href="#">Join Our Art Exhibition</a>
       </div>
       
     </section>
 
-    <section class="mt-4 mb-5 w-100 bg-light">
+    <section class="mt-4 mb-2 w-100 speaker-details">
       <div class="title d-flex flex-column justify-content-center">
         <h3 class="text-dark h4 fw-bold mt-4  text-center">Featured Speakers</h3>
         <hr class="hr mx-auto border mb-3 border-1 border-danger">
       </div>
 
       <!-- dynamic speakers data from app.js -->
-      <div class="speakers d-flex flex-column flex-md-row justify-content-between mx-md-auto px-md-5
-        mt-5 flex-md-wrap " id="speakers-section"></div>         
+      <!-- mobile version -->
+      <div class="speakers h-100 d-flex d-md-none flex-column flex-md-row justify-content-center mx-md-auto px-md-5
+        mt-2 flex-md-wrap pt-4" id="speakers-section"></div>
+      <!-- desktop version -->
+      <div class="speakers d-none d-md-flex gap-1 flex-column flex-md-row justify-content-center mx-md-auto px-md-5
+        mt-5 flex-md-wrap " id="desktop-speakers-section"></div>
```

Comment:
- [ ] [OPTIONAL] In the mobile version the square checks only appear on the first two speakers, when you click more button the rest of the speaker don’t have the square checks. Several factors such as consistency, colours, typography, imagery, simplicity, and functionality contribute to good website design. And a good ✅ website design makes for better user experience. And as these projects will be showcased in your resume and profile, its going to leave bad impression on the recruiters if the design is inconsistent

Your design:

<img width="133" alt="Screenshot 2023-03-16 at 11 05 28 AM" src="https://user-images.githubusercontent.com/44798044/225539847-3586e5f5-eee5-474a-becf-62a3ef20b969.png">